resources/views/layout/dashboard.blade.php

```diff
@@ -62,4 +62,7 @@
 </body>
 @yield('js')
 <script src="{{ mix('dist/js/all.js') }}"></script>
+<script type="text/javascript">
+    window.apiKey = "{{ auth()->user()->api_key }}";
```

Comment:
Part 2 of the workaround in `resources/assets/js/cachet.js`. My preference is to get rid of this change.